specification/keyvault/resource-manager/readme.md

```diff
@@ -218,11 +218,24 @@ input-file:
 directive:
 - suppress:
     - R3016 # The 'release_policy' property for KeyCreateParameters does not support camelCase.
+- suppress:
     - R3026 # The 'PrivateEndpointConnection' and 'PrivateLinkResource' sub-resources don't define PATCH as per Network Team's specification.
+- suppress:
     - R3025 # The 'PrivateLinkResource' is only accessible via List operation; does not define GET as per Network Team's specification.
+- suppress:
     - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
+- suppress:
     - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
+- suppress:
     - R4009 # Vault object is the only one that need to be tracked with SystemData
+- suppress: INVALID_REQUEST_PARAMETER
```

Comment:
The yaml title has typo. It should be:

### Supression

Reply:
You mean, update suppress to supression?